Cargo.toml

```diff
@@ -11,13 +11,19 @@ repository = "https://github.com/imager-io/ffmpeg-dev-rs"
 readme = "README.md"
 exclude = ["assets", "examples"]
 
+[features]
+default = []
+gpl = []
+x264 = ["x264-dev"]
+
 [dependencies]
 libc = "^0.2"
 num_cpus = "1.11.1"
+x264-dev = { path = "../x264-dev", optional = true }
```

Comment:
This relies on unmerged/unreleased changes upstream in https://github.com/imager-io/x264-dev/pull/1 and needs to be pointed at a released version before we can merge this PR